ee/session_recordings/session_summary/templates/identify-objectives/prompt.djt

```diff
@@ -0,0 +1,303 @@
+- $event_type: Type of interaction (e.g., click, submit) - specifies how the user interacted
+- event_id: Unique identifier for the event - ALWAYS use this to reference specific events, NEVER generate or modify event IDs
+- event_index: Index of the event in the session - helps understand sequence
+
+Use these events to reconstruct the user journey. Events are provided in a chronologically order. Focus on what these events reveal about user intentions rather than the raw event data itself.
+
+Don't include raw `elements_chain_texts` data in your summary, but use it to improve your understanding of user goals.
+</events_input_format>
+
+<events_input>
+```
+{{ EVENTS_DATA|safe }}
+```
+</events_input>
+
+<url_mapping_input_format>
+URLs mapping table shows the actual URLs for simplified URL references in the events data.
+
+When analyzing events, use these mappings to understand the actual pages visited. For example, if an event shows 'url_1', refer to this mapping to find the actual URL. In your summary and tags, always use the actual page/feature names, not the simplified URL references. Always refer to the URL with the simplest version e.g. `posthog.com` or `posthog.com/replay`, instead of mentioning the full URL with long query parameters.
+</url_mapping_input_format>
+
+<url_mapping_input>
+```
+{{ URL_MAPPING|safe }}
+```
+</url_mapping_input>
+
+<window_mapping_input_format>
+Window IDs mapping table shows the actual browser windows/tabs IDs for simplified references in the events data.
+
+Window IDs help track user activity across different browser windows or tabs. For example:
+
+- If events switch from 'window_1' to 'window_2', the user switched to a different browser window/tab
+- Multiple events with the same window ID indicate continuous activity in the same window/tab
+- Frequent window switches might indicate comparing content or multitasking
+
+Use this information to understand user navigation patterns and parallel browsing behavior.
+</window_mapping_input_format>
+
+<window_mapping_input>
+```
+{{ WINDOW_ID_MAPPING|safe }}
+```
+</window_mapping_input>
+
+<session_metadata_input_format>
+Use session metadata to understand user engagement context:
+
+- active_seconds: Total time user actively engaged with the application
+- inactive_seconds: Periods when user wasn't interacting with the application
+- click_count, keypress_count, mouse_activity_count: Interaction density
+- start_url: Entry point to the application, often indicates initial user intent
+
+Use these metrics to calibrate your analysis of user objectives and engagement quality.
+</session_metadata_input_format>
+
+<session_metadata_input>
+```
+{{ SESSION_METADATA|safe }}
+```
+</session_metadata_input>
+
+<identify_objectives_instructions>
+CRITICAL WARNING: DO NOT hallucinate errors. Only mark events as errors when there is EXPLICIT evidence in the event data that an error occurred.
```

Comment:
> CRITICAL WARNING: DO NOT hallucinate errors.

ah man, the world of prompt engineering is wild 🤣 😅

Reply:
Laugh as you want, but it was a drastical change 😄